.babelrc

```diff
@@ -1,4 +1,11 @@
 {
-	presets: ['es2015', 'react'],
-	plugins: ['transform-object-rest-spread']
-}
+	presets: [
+		'@babel/preset-env',
+		'@babel/preset-react',
+		'@babel/flow',
```

Comment:
It looks like `@babel/flow` helps with type checking? Would this be helpful on Encode as well?

Reply:
Maybe. We can discuss it.